lite-core/src/main/kotlin/org/taktik/icure/ICureBackendApplication.kt

```diff
@@ -195,6 +194,15 @@ class ICureBackendApplication {
             return  firstAttempt || secondAttempt || thirdAttempt
         }
 
+        suspend fun canWarmUp(
```

Comment:
maybe just change the name of this method to warmupPartitionAndCheckForCompletion ?